Gemfile

```diff
@@ -11,6 +11,8 @@ gem 'jquery-ui-rails', rails_version[0] == '3' ? '~> 4.0' : '~> 5.0'
 
 gem 'test-unit', '~> 3.0' if rails_version[0] == '3'
 
+gem 'arbre', github: 'activeadmin/arbre'
```

Comment:
[This change](https://github.com/activeadmin/arbre/pull/49) required, but new arbre version is not published